raiden/storage/sqlite.py

```diff
@@ -138,7 +138,7 @@ def count_state_changes(self) -> int:
         return int(query[0][0])
 
     def write_state_change(self, state_change, log_time):
-        with self.write_lock, self.conn:
+        with self.write_lock:
```

Comment:
Add `maybe_commit` after `executemany` here and in other places where self.conn context was removed